.github/workflows/release.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           git config --global user.email "vhdl-linter-bot@niklas-schelten.de"
       - name: Get Current Version Number
         run: |
-          CURRENT_VERSION=$(cat package.json | jq .version | cut -d'"' -f 2)
+          CURRENT_VERSION=$(npx vsce show g0t00.vhdl-linter --json | jq -r '.versions[0].version')
```

Comment:
Maybe use CURRENT_VERSION for the changelog as well?

Reply:
This would fail if the last tag has not been published. Don't know what is better tbh

Reply:
Hm? I was proposing the opposite, showing the changes in the changelog between new version and the last released (to vscode) version.